lib/matplotlib/figure.py

```diff
@@ -710,7 +714,9 @@ def set_size_inches(self, w, h=None, forward=True):
         # argument, so unpack them
         if h is None:
             w, h = w
-
+        if not all(np.isfinite(_) for _ in (h, w)):
```

Comment:
Really picky, but it's kind of weird to have it be `h, w` when everything around it is `w, h`.